### ISSUE TITLE
Feat: Dark Mode for the App

### ISSUE THREAD

**Issue by NupurThakur27** [nic-delhi/AarogyaSetu_Android#292]
**Current Behavior**
Currently the App doesn't have Dark mode.

**Desired Behavior**
AarogyaSetu App should support dark mode for the UI benefits.

---
**Resources**
https://blog.mindorks.com/implementing-dark-mode-theme-in-android

**Comment by vinaybedre** [comment]
I appreciate Dark is easy on eyes, provided you use the app for a longer time (eg. WhatsApp/Facebook or any news reader apps). 
Its not a must for an app like AarogyaSetu

**Comment by NupurThakur27** [comment]
> I appreciate Dark is easy on eyes, provided you use the app for a longer time (eg. WhatsApp/Facebook or any news reader apps).
> Its not a must for an app like AarogyaSetu

I completely agree that it's not must but features like dark modes help some user to attract towards the app as Currently in this time of uncertainty, we don't know how long we have to stay with COVID-19, We should also look at attracting all kinds of the audience.
P.S.: I totally agree that this feature is not a "must".

**Comment by adityaruplaha** [comment]
Would be nice to have, but this is not the highest priority I would say.

**Comment by sanabana** [comment]
We need to check for the vulnerabilities in the app.We need to find bug not features.Talking about features is really good but try to focus on finding bugs.

**Comment by hardik0899** [comment]
Dark mode saves battery.. So at some later point of time when all bugs are solved this feature could be taken up in my opinion..

**Comment by Rishika1416** [comment]
we are a team of developers , we would like to add this dark mode to the app . Can we proceed?
